ui/src/app/pages/homepage/recent-workspaces.tsx

```diff
@@ -42,22 +41,30 @@ export const RecentWorkspaces = withUserProfile()
 
   render() {
     // Needs a min-height so the spinner will render when loading and position: relative so said spinner will center.
-    return <FlexRow style={{marginTop: '1rem', minHeight: 247, position: 'relative'}}>
-      {
-        !this.state.loading && this.state.recentWorkspaces.length === 0 && <NewWorkspaceButton />
-      }
-      {
-        this.state.recentWorkspaces.map(recentWorkspace => {
-          return <WorkspaceCard
-            key={recentWorkspace.workspace.name}
-            userEmail={this.props.profileState.profile.username}
-            workspace={recentWorkspace.workspace}
-            accessLevel={recentWorkspace.accessLevel}
-            reload={() => this.loadWorkspaces()}
-          />;
-        })
-      }
-      {this.state.loading && <SpinnerOverlay dark={true} />}
-    </FlexRow>;
+    return <React.Fragment>
+        {!this.state.loading && this.state.recentWorkspaces.length === 0 ? (
+          <div style={{color: colors.primary, padding: '10px 30px'}}>
```

Comment:
maybe this isn't the best place for putting the padding? We can add padding in the parent div of RecentWorkspaces so that the parent is aware of how much padding it needs.

Reply:
The parent div won't work in this case unless I add a new div in between. The parent div contains the Workspace header which does not need the padding.

Is there an anti pattern with padding your own content?

Reply:
Yeah I think you still want margin here. I'm realizing it probably doesn't need to be in a parent element.

More info: https://stackoverflow.com/questions/2189452/when-to-use-margin-vs-padding-in-css